diff --git a/index.php b/index.php
index 25905e2..ac862f4 100644
--- a/index.php
+++ b/index.php
@@ -1,7 +1,8 @@
 <?php
-$walker1 = 'Kitty';
-$walker2 = 'Tiger';
-$walker3 = 'Jay';
+$waggyPig = ['Waggy Pig', 10, 7, 'Sleepy white fluffy dog'];
+?>
 
-$dogWalkers = [$walker1, $walker2, $walker3];
-var_dump($dogWalkers);
+<h2><?php echo $waggyPig[0]; ?></h2>
+<div class="age"><?php echo $waggyPig[2]; ?></div>
+<div class="weight"><?php echo $waggyPig[1]; ?></div>
+<p><?php echo $waggyPig[3]; ?></p>
