diff --git a/index.php b/index.php
index 3c9c10d..b604ded 100644
--- a/index.php
+++ b/index.php
@@ -1,11 +1,11 @@
+<h4>
+    <!-- replace the XXXX with the real number of toys using count -->
+    Selling XXXX Toys
+</h4>
+
 <?php
 $toys = [];
 $toys[] = ['name' => 'Bacon Bone', 'color' => 'Bacon-colored'];
 $toys[] = ['name' => 'Tennis Ball', 'color' => 'Yellow'];
 $toys[] = ['name' => 'Frisbee', 'color' => 'Red'];
 ?>
-
-<?php foreach ($toys as $toy) { ?>
-    <h3><?php echo $toy['name']; ?></h3>
-    <h4><?php echo $toy['color']; ?></h4>
-<?php } ?>
