diff --git a/index.php b/index.php
index 25d34e8..f0ccf4e 100644
--- a/index.php
+++ b/index.php
@@ -1,8 +1,13 @@
 <?php
-$waggyPig = array('Waggy Pig', 10, 7, 'Sleepy white fluffy dog');
+$waggyPig = array(
+    'name' => 'Waggy Pig',
+    'weight' => 10,
+    'age' => 7,
+    'bio' => 'Sleepy white fluffy dog',
+);
 ?>
 
-<h2><?php echo $waggyPig[0]; ?></h2>
-<div class="age"><?php echo $waggyPig[2]; ?></div>
-<div class="weight"><?php echo $waggyPig[1]; ?></div>
-<p><?php echo $waggyPig[3]; ?></p>
+<h2><?php echo $waggyPig['name']; ?></h2>
+<div class="age"><?php echo $waggyPig['age']; ?></div>
+<div class="weight"><?php echo $waggyPig['weight']; ?></div>
+<p><?php echo $waggyPig['bio']; ?></p>
