diff --git a/index.php b/index.php
index a687aff..1c4e953 100644
--- a/index.php
+++ b/index.php
@@ -7,9 +7,9 @@ $toys = json_decode($contents, true);
     <h3><?php echo $toy['name']; ?></h3>
     <h4>
         <?php
-        if (array_key_exists('color', $toy) && $toy['color'] = 'surprise') {
+        if (array_key_exists('color', $toy) && $toy['color'] == 'surprise') {
             echo 'Surprise Color!';
-        } elseif !array_key_exists('color', $toy) {
+        } elseif (!array_key_exists('color', $toy)) {
             echo 'no color';
         } else {
             echo $toy['color'];
