diff --git a/index.php b/index.php
index 80da7ea..0fd577b 100644
--- a/index.php
+++ b/index.php
@@ -5,5 +5,11 @@ $toys = json_decode($contents, true);
 
 <?php foreach ($toys as $toy) { ?>
     <h3><?php echo $toy['name']; ?></h3>
-    <h4><?php echo $toy['color']; ?></h4>
+    <h4>
+        <?php if (array_key_exists('color', $toy)) { ?>
+            <?php echo $toy['color']; ?>
+        <?php } else { ?>
+            no color
+        <?php } ?>
+    </h4>
 <?php } ?>
