diff --git a/index.php b/index.php
index 0fd577b..c9045aa 100644
--- a/index.php
+++ b/index.php
@@ -6,10 +6,10 @@ $toys = json_decode($contents, true);
 <?php foreach ($toys as $toy) { ?>
     <h3><?php echo $toy['name']; ?></h3>
     <h4>
-        <?php if (array_key_exists('color', $toy)) { ?>
-            <?php echo $toy['color']; ?>
-        <?php } else { ?>
-            no color
-        <?php } ?>
+        <?php
+        if (array_key_exists('color', $toy)) {
+            echo $toy['color'];
+        }
+        ?>
     </h4>
 <?php } ?>
diff --git a/toys.json b/toys.json
index 926a2b8..929b75e 100644
--- a/toys.json
+++ b/toys.json
@@ -8,6 +8,6 @@
     },
     {
         "name": "Frisbee",
-        "color": "Red"
+        "color": "multiple"
     }
 ]
