diff --git a/index.php b/index.php
index ad0467b..80da7ea 100644
--- a/index.php
+++ b/index.php
@@ -1,7 +1,9 @@
 <?php
-file_put_contents('doglife.txt', 'Dogs rule!');
+$contents = file_get_contents('toys.json');
+$toys = json_decode($contents, true);
 ?>
 
-<h2>
-    <?php echo file_get_contents('doglife.txt'); ?>
-</h2>
+<?php foreach ($toys as $toy) { ?>
+    <h3><?php echo $toy['name']; ?></h3>
+    <h4><?php echo $toy['color']; ?></h4>
+<?php } ?>
diff --git a/toys.json b/toys.json
index f04f7ce..926a2b8 100644
--- a/toys.json
+++ b/toys.json
@@ -1,7 +1,6 @@
 [
     {
-        "name": "Bacon Bone",
-        "color": "Bacon-colored"
+        "name": "Bacon Bone"
     },
     {
         "name": "Tennis Ball",
