diff --git a/new_toy.php b/new_toy.php
index 7722d15..9e887e0 100644
--- a/new_toy.php
+++ b/new_toy.php
@@ -1,8 +1,10 @@
 <?php
-$name = $_POST['name'];
-$description = $_POST['description'];
+if ($_SERVER['REQUEST_METHOD'] == 'POST') {
+    $name = $_POST['name'];
+    $description = $_POST['description'];
 
-var_dump($name, $description);
+    var_dump($name, $description);
+}
 ?>
 
 <form action="/new_toy.php" method="POST">
@@ -10,4 +12,4 @@ var_dump($name, $description);
     <textarea name="description"></textarea>
 
     <button type="submit">Add toy</button>
-</form>
+</form>
\ No newline at end of file
