diff --git a/new_toy.php b/new_toy.php
index 9e887e0..3fcd4bd 100644
--- a/new_toy.php
+++ b/new_toy.php
@@ -1,15 +1,3 @@
-<?php
-if ($_SERVER['REQUEST_METHOD'] == 'POST') {
-    $name = $_POST['name'];
-    $description = $_POST['description'];
-
-    var_dump($name, $description);
-}
-?>
-
-<form action="/new_toy.php" method="POST">
-    <input type="text" name="name" />
-    <textarea name="description"></textarea>
-
-    <button type="submit">Add toy</button>
-</form>
\ No newline at end of file
+<h3>
+    Print the browser information of your user here
+</h3>
